quarkus-integration/quarkus/devui-integration-test/src/test/java/ai/timefold/solver/quarkus/it/devui/TimefoldDevUITest.java

```diff
@@ -57,51 +46,28 @@ void testSolverConfigPage() throws ParserConfigurationException, SAXException, I
     }
 
     @Test
-    void testModelPage() throws ParserConfigurationException, SAXException, IOException {
-        String body = RestAssured.get(getPage("model"))
-                .then()
-                .extract()
-                .body()
-                .asPrettyString();
-        XmlParser xmlParser = new XmlParser();
-        Node node = xmlParser.parseText(body);
-        String model = Objects.requireNonNull(findById("timefold-solver-model", node)).toString();
-        assertThat(model)
-                .contains("value=[Solution: ai.timefold.solver.quarkus.it.devui.domain.TestdataStringLengthShadowSolution]");
-        assertThat(model)
-                .contains("value=[Entity: ai.timefold.solver.quarkus.it.devui.domain.TestdataStringLengthShadowEntity]");
-        assertThat(model).contains(
-                "value=[Genuine Variables]]]]]], tbody[attributes={}; value=[tr[attributes={}; value=[td[attributes={colspan=1, rowspan=1}; value=[value]]");
-        assertThat(model).contains(
-                "value=[Shadow Variables]]]]]], tbody[attributes={}; value=[tr[attributes={}; value=[td[attributes={colspan=1, rowspan=1}; value=[length]]");
+    void testModelPage() throws Exception {
+        JsonNode modelResponse = super.executeJsonRPCMethod("getModelInfo");
+        assertThat(modelResponse.get("solutionClass").asText())
+                .contains("ai.timefold.solver.quarkus.it.devui.domain.TestdataStringLengthShadowSolution");
```

Comment:
I'd use the `class.getCanonicalName()` throughout this test, to make this safe for refactoring.